test/utils/shippable/sanity.sh

```diff
@@ -13,18 +13,32 @@ if [ "${install_deps}" != "" ]; then
     apt-get update -qq
     apt-get install shellcheck
 
+    # FIXME REPOMERGE, remove the following
     pip install git+https://github.com/ansible/ansible.git@devel#egg=ansible
```

Comment:
In a way it is required. We need a copy of Ansible before and after. This is how we detect new modules. By effectively checking if the module in the PR is in the installed version of Ansible.

There may be other ways to work around this, but that would have to be solved.